anabot/runtime/installation/hub/software_selection.py

```diff
@@ -216,3 +223,49 @@ def addon_check(element, app_node, local_node):
     if action_result(element)[0] == False:
         return action_result(element)
     return addon_handler_manipulate(element, app_node, local_node, True)
+
+def get_full_kernel_option_name(kernel_option):
+    if kernel_option == '4k':
+        return FULL_4k_KERNEL_OPTION
+    elif kernel_option == '64k':
+        return FULL_64k_KERNEL_OPTION
+    else:
+        return None
+
+@handle_act('/kernel_options')
+def kernel_options_handler(element, app_node, local_node):
+    chosen_kernel_option = get_attr(element, "value")
+    if chosen_kernel_option is None:
+        return (False, "No kernel options specified")
+    try:
+        kernel_combo_box = getnode(local_node, "combo box",
+                                   tr(f"{DEFAULT_KERNEL_OPTION}", context="GUI|Software Selection|Kernel Options"))
+    except TimeoutError:
+        return (False, f"Couldn't find the {DEFAULT_KERNEL_OPTION} 'Kernel Options combo box'")
+    kernel_combo_box.click()
+    try:
+        kernel_option_window = getnode(app_node, "window")
+    except TimeoutError:
+        return (False, "Couldn't find the opened 'Kernel Options' window")
+    kernel_option_full_name = get_full_kernel_option_name(chosen_kernel_option)
+    try:
+        kernel_option_menu_item = getnode(kernel_option_window, "menu item",
+                                          tr(f"{kernel_option_full_name}", context="GUI|Software Selection|Kernel Options"))
+    except TimeoutError:
+        return notfound('"%s"' % chosen_kernel_option, where="Kernel Options combo box")
+    kernel_option_menu_item.click()
+    return True
```

Comment:
Rather use `ActionResultPass()` in new code, please (also in other occurrences).

Reply:
Thanks for the suggestion.